proposals/working-group/Comparisons.adoc

```diff
@@ -0,0 +1,36 @@
+= Pros/Cons of Each Proposal
+
+== MPWG
+Pros
+
+* Flatter governing structure as steering committee, marketing committee and specification committees share the same membership.
+* Addresses IP gap in existing MP development
+* Provides complete autonomy of how MP evolves as governing structure is not part of a non-MP specific working group
```

Comment:
How is this NOT achieved in the CN4J working group?

Reply:
@smillidge Good question.

- The MPWG proposal intends to flatten the Steering Committee and Specification committee (all members of one are also in the other). MicroProfile cannot flatten the organizational structure if there is a shared steering committee under CN4J. 
- The "pro" here just calls out that the IP gap, the primary motivation behind the creation of a WG, is addressed
- CN4J does not provide for complete autonomy if there is a shared steering committee. CN4J including both MicroProfile and Jakarta EE is a  starting point, not a destination. The intent is to open up CN4J to be a landing place for additional relevant cloud native projects in the future. That means that WG members that join to influence non-MicroProfile projects could also influence MicroProfile. The reverse is true, in that any member that joins to influence non-Jakarta EE projects would also have influence over Jakarta EE.